src/screens/Connectors/PaymentProcessor/ConnectorHome.res

```diff
@@ -1,6 +1,6 @@
 module ConnectorCurrentStepIndicator = {
   @react.component
-  let make = (~currentStep: ConnectorTypes.steps, ~stepsArr, ~borderWidth="w-8/12") => {
+  let make = (~currentStep: ConnectorTypes.steps, ~stepsArr, ~borderWidth="w-9/12") => {
```

Comment:
this `borderWidth` is not getting used anywhere in the component, it can be removed from the props